SR2024/2023-12-13-challenge-reminder.md

```diff
@@ -0,0 +1,15 @@
+Hi $TODO_CONTACT_GIVEN_NAME,
+
+Just a reminder that the first challenge submission deadline passed over the weekend. 
```

Comment:
This opening sentence feels confusing to me. Starting out as a reminder sounds like it's going to be something the teams may need to do something about, yet it being in the past negates that.

Perhaps opening with the "Congratulations" line would be better?

```suggestion
Congratulations to the 15 teams that completed one of the challenges in time for the first submission deadline this past weekend. We've been really impressed with the challenge submissions this year, it's been great to see the creativity of so many teams so early on in the competition.
```

(And then obviously drop that line from below)